.gitignore

```diff
@@ -0,0 +1 @@
+./spec/spec_helper.rb
```

Comment:
You didn't upload the spec_helper to github and is required to load your tests.
You can either delete it from the gitignore of from the next file I commented on.